docs/scenarios_authoring/Fragments.md

```diff
@@ -9,14 +9,34 @@ Fragments are a way to share processing logic - if many scenarios require identi
 Once a fragment is created using Designer (see Fragments tab) it can be used in other scenarios in the same category.
 
 ## Inputs
-Fragment can have one input. You can define parameters of a fragment:
+Fragment must have exactly one input. You can define parameters of a fragment:
 
-![fragment input](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_fragments_-_Inputs0.png)
+![fragment input_base](img/fragment_input_simple.png)
+
+Clicking the "Show advanced configuration" button opens up an additional form, that allows changing the parameter's behaviour during inputting its value when using the fragment, e.g. changing its validation requirements or displaying a hint.
```

Comment:
I would simplify this sentence to just 
... _additional form, that allow to set additional options for the input parameters.

You can see options available in the screenshot, so I think no need to enumerate them here. Less text is good to me.